ietf/bin/rfc-editor-queue-updates

```diff
@@ -21,9 +20,11 @@ from ietf.utils.log import log
 
 log("Updating RFC Editor queue states from %s" % settings.RFC_EDITOR_QUEUE_URL)
 
-socket.setdefaulttimeout(30)
-response = requests.get(settings.RFC_EDITOR_QUEUE_URL).text
-drafts, warnings = parse_queue(io.StringIO(response))
+response = requests.get(
+    settings.RFC_EDITOR_QUEUE_URL,
+    timeout=30,  # seconds
+)
```

Comment:
The addition of the timeout opens the possibility of raising a requests.exceptions.Timeout exception. It would be nice to handle that, though it would be understandable if you decide that not erroring out of the command line utility is a correct behavior.